src/components/Rectangle/AppImages/index.jsx

```diff
@@ -8,11 +8,13 @@ import styles from './index.scss';
 
 export default class AppImages extends Component {
   static propTypes = {
-    apps: PropTypes.array
+    apps: PropTypes.oneOfType([PropTypes.object, PropTypes.array]),
+    total: PropTypes.oneOfType([PropTypes.number, PropTypes.string])
```

Comment:
why add `total` prop? Does this prop can be produced by `apps`?

Reply:
total is total_count of api result , apps is some ahead apps  of category apps.